### DISCUSSION TITLE
Quarto presentation - two column layout

### DISCUSSION THREAD

**Discussion by Andrzej-Andrzej** [quarto-dev/quarto-cli#5726] [Q&A] [answered]
### Description

Hi,
I would like to ask for help regarding reveal.js presentation. I want to make Quarto presentation with 2 column layout (landscape) so in 1-st column I will have a code and in the second column there will be placed output of that code. I would like to do it for all slides. Just under those 2 columns I will place plain text. How to achieve this ?

**Comment by mcanouil** [comment, marked as answer]
<img width="1624" alt="Screenshot 2023-05-29 at 14 43 40" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5e379fcf-6caa-4509-99bc-9b9d71dcfb8c">

Please consider searching in the documentation before asking question.
If you don't find what you are looking for in the documentation, or if it's not clear, please do give us feedbacks on how to improve the documentation.

GitHub Issues and Discussions are not a replacement of the search engine in the documentation website.
I hope you understand the difference.

**Reply by Andrzej-Andrzej** [reply]
Thank you, I try to google it or find it anyway. 
Maybe you find this question valid:
"I would like to kindly ask as well, how do I know being in Quarto file, which slide am I working on ?
Should I manually number them ? Is it an automatic method for it ? 
When rendered, there are numbers at the bottom right corner of the slide of course, but how do I find myself eg. when I want to go to slide 45 in Quarto file and add a line of text or code to it ?"
When working with Powerpoint this is easy to see, but when I am working with qmd file and have large alount of slides this is difficult to jump between slides. Maybe this would be good to mention in documentation.

**Reply by mcanouil** [reply]
> Thank you, I try to google it or find it anyway.

Go on the website and use the search module. There is no need for outside search engine.

> I would like to kindly ask as well, how do I know being in Quarto file, which slide am I working on ?
Should I manually number them ? Is it an automatic method for it ?
When rendered, there are numbers at the bottom right corner of the slide of course, but how do I find myself eg. when I want to go to slide 45 in Quarto file and add a line of text or code to it ?

No tools for that at the moment.
Use your IDE "outline" feature (both RStudio/VSCode IDE has it) to find the sections.

Split your large presentation into smaller pieces and use the include shortcode.
